### DISCUSSION TITLE
Wrong display of files in the status modal

### DISCUSSION THREAD

**Discussion by h7b** [ObsidianPublisher/obsidian-github-publisher#173] [Q&A — Plugin] [answered]
Hi Mara,

I found a small confusion in the modal which display the status of files.

As you can see in screencap, these 3 files were edited. But they were classified in the `Added` section, instead of the `Edited` section. Is this something wrong on my end, or it's just a bug?

![Screenshot 2023-07-02 120241](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/86811584/7e5e9cdc-d731-4250-9b09-0372551e47db)

**Comment by Mara-Li** [comment]
Added : the file don't exists on the repo
Edited : The file existed and was updated

**Reply by h7b** [reply]
exactly it's what I expected. But the modal displayed the result in wrong section.

Here what I did to reproduce
- I edit the file `cmt-lvl1-2020.md` again (which were already uploaded into the repo as in the previous screencap)
- use command `Refresh published and upload new notes`
- then the file `cmt-lvl1-2020.md` is listed in the `Added` section, instead of `Edit` section

Hope you get what I meant.

![Screenshot 2023-07-02 123317](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/86811584/1787013b-6717-42f0-adfb-d80eaaf07b48)

**Comment by Mara-Li** [comment, marked as answer]
Oh my god.
I'm so stupid, I inverted the added/edited in the code!
Good catch :'D
